PennMobile/Courses/Models/CoursesViewModel.swift

```diff
@@ -32,25 +32,25 @@ private func getTimeInt(pathAtPennString: String) -> Int? {
 extension Course {
     /// Initializes a course from Path@Penn data.
     init(_ data: PathAtPennNetworkManager.CourseData) {
-        var crn = data.crn
-        var code = data.code
-        var title = data.title
-        var section = data.section
+        let crn = data.crn
+        let code = data.code
+        let title = data.title
```

Comment:
Lol I have a branch where I'm fixing the warnings and removing iOS 15 support and I did this same thing